### DISCUSSION TITLE
Layout switching for Basic controls

### DISCUSSION THREAD

**Discussion by Tak-attack** [redphx/better-xcloud#244] [General]
Now that the ability to switch between custom layouts has been implemented, could it be possible to replace the official basic controls layout and switch between multiple layouts?

If there were layouts for different game genres, such as RPG, racing, FPS, twin stick shooter, one-handed controls, etc., it would save the time and effort of creating custom controls for each game.

I am a complete amateur, so I don't know if this is feasible.

This is just an idea suggestion, so it can be low priority.

**Comment by redphx** [comment]
I had the same idea, but I think it's difficult to create one layout that can be applied to multiple games in the same genres (since the buttons & binding are different between each games). So it's better to copy/paste and modify for each game.

Xbox has some sample layouts for each genre: https://github.com/microsoft/xbox-game-streaming-tools/tree/main/touch-adaptation-kit/samples/sample-layouts

**Comment by bah99** [comment]
Hi, I have made a highly customizable virtual controller userscript for fps games https://gist.github.com/bah99/2ab1ac27555089247ed3b0220d1e9ea7
